src/components/pages/devices/flyouts/deviceDelete/deviceDelete.js

```diff
@@ -61,13 +67,49 @@ export class DeviceDelete extends Component {
     if (this.subscription) this.subscription.unsubscribe();
   }
 
+  /*
+  Because the ag-grid doesn't update its state for selected items when something is removed from its props/rowData,
+  we need to take the props.devices we get here and ensure it still exists in the redux store.
+  */
+  ensureDevicesExist = (devices = []) => {
+    return devices.reduce(
```

Comment:
Use the implicit return

Reply:
Yeah, yeah. Just a PITA to convert every time ... when debugging and having to use console.log to do it.